indico/modules/attachments/templates/mako_compat/attachments_tree.html

```diff
@@ -0,0 +1,16 @@
+{% from 'attachments/_attachments.html' import render_attachments %}
+
+<div class="attachments-box {%- if can_edit %} highlighted-area{% endif %}">
+    <div class="attachments-box-header">
+        <h2 class="icon-attachment"> {% trans %}Materials{% endtrans %}</h2>
```

Comment:
In that case you might also  want to update the two places where it's currently being used in _attachments.html